app/src/main/java/ch/epfl/sdp/drone3d/storage/data/LatLong.kt

```diff
@@ -0,0 +1,3 @@
+package ch.epfl.sdp.drone3d.storage.data
+
```

Comment:
Couldn't we use class LatLng from mapbox (com.mapbox.mapboxsdk.geometry.LatLng) directly instead of creating our own class? Might be easier to use with the map (no adaptation needed) and some functions are already implemented.

Reply:
To be able to use the firebase serialiser, all stored object  need to have an empty constructor. That's why we choose the redefine our own LatLng.